compiler/src/iree/compiler/Codegen/Common/TileDispatchUsingForall.cpp

```diff
@@ -33,8 +33,31 @@ namespace {
 struct TileAndDistributeToWorkgroupsUsingForallOpPass final
     : public impl::TileAndDistributeToWorkgroupsUsingForallOpPassBase<
           TileAndDistributeToWorkgroupsUsingForallOpPass> {
+  TileAndDistributeToWorkgroupsUsingForallOpPass(bool strategy)
```

Comment:
Because we might want more strategies in the future - or at least it feels like we might, can we make this an enum? `cl` has options for defining enums

Reply:
Thanks. I'll make sure to add that.

Reply:
After reviewing the reorderWorkgroup pass and implementing it here, there is only one strategy (transpose) available, which I have added. (Moved to boolean)